application/include/CameraView.h

```diff
@@ -0,0 +1,21 @@
+#ifndef _CameraView_H_
+#define _CameraView_H_
+
+#include <SFML/Graphics.hpp>
+#include 'GameLogic.h'
+
+class CameraView {
+  public:
+    CameraView();
+    void init();
+    void draw(sf::RenderWindow&); 
+    void switchScreen(int);
+  private:
+    sf::Font font;
+    GameLogic logic;
+    //other drawable objects??
```

Comment:
My initial thoughts on this were to query the objects from the GameLogic and then draw them:
`sf::Vector2f player1_pos = this -> logic -> player1.getPos();`
`circle.setPosition(player1_pos);`
Maybe there's a better way, what do you think?